internal/tools/pipeline/pipengine/pvolumes/bind.go

```diff
@@ -53,7 +55,17 @@ func GenerateTaskCommonBinds(mountPoint string) []apistructs.Bind {
 		ContainerPath: dockerSock,
 		ReadOnly:      true,
 	}
-	binds = append(binds, dockerSockBind)
+	defaultContainerdSockBind := apistructs.Bind{
+		HostPath:      defaultContainerdSock,
```

Comment:
How about path not exists?

Reply:
never mind, if hostpath not exists, just create a empty directory

[reference](https://github.com/kubernetes/kubernetes/issues/52318)